package.json

```diff
@@ -30,7 +30,8 @@
         "build": "run-s lint docs",
         "debug": "node --nolazy --inspect-brk=9229 lib/index.js",
         "docs": "jsdoc2md --files lib/**/*.js > docs/dist/documentation.md",
-        "lint": "eslint --fix lib/**/*.js && eslint --fix test/**/*.js && prettier --write lib/**/*.js",
+        "lint:fix": "eslint --fix lib/**/*.js && eslint --fix test/**/*.js && prettier --write lib/**/*.js",
+        "lint": "eslint lib/**/*.js && eslint test/**/*.js",
```

Comment:
why the change? for the commit hook?

Reply:
because lint shouldnt fix, it should lint only. This way we can run it in a check and not just hide issues